frontend/src/utilities/useAcceleratorProfileFormState.ts

```diff
@@ -27,64 +27,110 @@ const useAcceleratorProfileFormState = (
   tolerations?: Toleration[],
   existingAcceleratorProfileName?: string,
   namespace?: string,
+  acceleratorProfileNamespace?: string,
 ): UseAcceleratorProfileFormResult => {
   const [globalScopedInitialState, globalScopedLoaded, globalScopedLoadError, refresh] =
     useReadAcceleratorState(resources, tolerations, existingAcceleratorProfileName);
   const [projectScopedInitialState, projectScopedLoaded, projectScopedLoadError] =
-    useReadAcceleratorState(resources, tolerations, existingAcceleratorProfileName, namespace);
+    useReadAcceleratorState(
+      resources,
+      tolerations,
+      existingAcceleratorProfileName,
+      namespace,
+      acceleratorProfileNamespace,
+    );
 
   const loaded = namespace ? projectScopedLoaded && globalScopedLoaded : globalScopedLoaded;
   const loadError = namespace
     ? projectScopedLoadError || globalScopedLoadError
     : globalScopedLoadError;
 
   const initialState = React.useMemo(() => {
+    // Check if project-scoped feature flag is off but workbench has project-scoped profile
+    // Only apply this if acceleratorProfileNamespace is NOT the global dashboard namespace
+    if (
+      !namespace &&
+      existingAcceleratorProfileName &&
+      acceleratorProfileNamespace &&
+      acceleratorProfileNamespace !==
+        globalScopedInitialState.acceleratorProfiles[0]?.metadata.namespace
+    ) {
+      // Feature flag is off (no namespace) but workbench has project-scoped profile
+      // Show "Existing settings" instead of searching for global profile with same name
+      const state: AcceleratorProfileState = {
+        acceleratorProfiles: globalScopedInitialState.acceleratorProfiles,
+        acceleratorProfile: undefined,
+        count: 1,
+        unknownProfileDetected: true,
+      };
+      return state;
+    }
+
     // If we have a namespace, use project-scoped state but include global profiles
     if (namespace) {
       // Keep project and global profiles separate
       const projectProfiles = [...projectScopedInitialState.acceleratorProfiles];
       const globalProfiles = [...globalScopedInitialState.acceleratorProfiles];
+      const allProfiles = [...projectProfiles, ...globalProfiles];
 
-      // Determine which accelerator profile to use
-      const acceleratorProfile =
-        projectScopedInitialState.acceleratorProfile || globalScopedInitialState.acceleratorProfile;
+      // Search for the existing accelerator profile by both name and namespace
+      let acceleratorProfile: AcceleratorProfileKind | undefined;
+      if (existingAcceleratorProfileName && acceleratorProfileNamespace) {
+        // Look for the profile by both name and namespace in all available profiles
+        acceleratorProfile = allProfiles.find(
+          (ap) =>
+            ap.metadata.name === existingAcceleratorProfileName &&
+            ap.metadata.namespace === acceleratorProfileNamespace,
+        );
```

Comment:
Here can we just use projectProfiles instead allProfiles, as we know that the annotation will present only for the project-scoped items?